packages/elements/src/components/ino-datepicker/ino-datepicker.tsx

```diff
@@ -180,6 +159,7 @@ export class Datepicker implements ComponentInterface {
 
   @Watch('inoDateFormat')
   inoDateFormatChanged(dateFormat: string) {
+    this.validator.dateFormat = dateFormat;
```

Comment:
Should we also validate if dateformat changes? Maybe there is a valid usecase for dynamic format changes